packages/embarkjs-swarm/CHANGELOG.md

```diff
@@ -0,0 +1,4 @@
+# [4.0.1](https://github.com/embark-framework/embark/compare/v4.0.0...v4.0.1) (2019-04-08)
```

Comment:
Should this file be removed if the plan is still to let lerna manage creation of changelogs for newly introduced packages?